operators/pkg/instanceset-landing/options.go

```diff
@@ -0,0 +1,63 @@
+package instanceset_landing
+
+import (
+	"errors"
+	"flag"
+	"strings"
+
+	"k8s.io/klog/v2"
+
+	clv1alpha2 "github.com/netgroup-polito/CrownLabs/operators/api/v1alpha2"
+)
+
+type options struct {
+	IdentifierHashKey string
+	HostName          string
+	DynamicStartup    bool
+	Template          clv1alpha2.GenericRef
+	CourseCode        string
+	Namespace         string
+	ListenerAddr      string
+}
+
+// Options object holds all the instanceset parameters.
+var Options options
+
+// Initialize flags and associate each parameter to the given options object.
+func (o *options) Init() {
+	flag.StringVar(&o.ListenerAddr, "address", ":8080", "[address]:port of the landing server")
+	flag.StringVar(&o.CourseCode, "course", "", "Course code")
+	flag.StringVar(&o.Template.Name, "template-name", "", "CrownLabs Template for the exam")
+	flag.StringVar(&o.Template.Namespace, "template-namespace", "workspace-exams", "Namespace of CrownLabs Template for the exam")
+	flag.StringVar(&o.Namespace, "namespace", "", "Custom namespace name, if not given it's automatically generated")
+	flag.StringVar(&o.HostName, "hostname", "crownlabs.polito.it", "CrownLabs instanceset host")
```

Comment:
Is this still required or could it be dropped?

Reply:
IIRC that was to give instances a custom hostname so that it can be selectively unlocked on LDB. It can be removed now and put back later if you think it's better

Reply:
Since we were planning to add that info to the template it would no longer be needed,